.eslintrc.js

```diff
@@ -145,21 +153,13 @@ module.exports = {
                     'error',
                     {
                         paths: [
-                            ...globalRestrictedImport,
-                            {
-                                name: 'lodash/get',
-                                message: 'Please use optional chaining and nullish coalescing instead.',
-                            },
-                            {
-                                name: 'lodash',
-                                importNames: ['get'],
-                                message: 'Please use optional chaining and nullish coalescing instead.',
-                            },
```

Comment:
why did we remove these restricted imports?

Reply:
I removed these as `you-dont-need-lodash-underscore` eslint rule handles such imports already:
![image](https://github.com/Expensify/App/assets/46095609/6f9b6a72-eb0a-4df5-8607-73d8e72d6e36)